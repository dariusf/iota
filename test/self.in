Person := Object clone
Person talk := method(writeln(name))
bob := Person clone
bob name := "bob"
bob talk