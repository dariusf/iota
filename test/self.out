Lobby.send('Lobby').send('setSlot', IoStringWrapper('Person'), Lobby.send('Object').send('clone'));
Lobby.send('Person').send('setSlot', IoStringWrapper('talk'), Lobby.send('method', IoThunk(function (locals) {
    return locals.send('writeln', locals.send('name'));
})));
Lobby.send('Lobby').send('setSlot', IoStringWrapper('bob'), Lobby.send('Person').send('clone'));
Lobby.send('bob').send('setSlot', IoStringWrapper('name'), IoStringWrapper('bob'));
Lobby.send('bob').send('talk');